sched/signal/sig_timedwait.c

```diff
@@ -319,7 +323,7 @@ int nxsig_timedwait(FAR const sigset_t *set, FAR struct siginfo *info,
            * time in nanoseconds.
            */
 
-#ifdef CONFIG_HAVE_LONG_LONG
+#ifdef CONFIG_SYSTEM_TIME64
           uint64_t waitticks64 = ((uint64_t)timeout->tv_sec * NSEC_PER_SEC +
                                   (uint64_t)timeout->tv_nsec +
                                   NSEC_PER_TICK - 1) /
```

Comment:
Most probably `waitticks64` now is not needed.

Reply:
Yes, but I think it's better to use `system_time64` here than `have long long`

Reply:
I mean that waitticks is now uint64_t, so temporary variable can be removed